datadog_checks_dev/datadog_checks/dev/tooling/commands/release/trello/tester_selector/github_users.py

```diff
@@ -52,8 +52,25 @@ def get_users(self, teams: List[str]) -> List[GithubUser]:
     def get_last_pr_date(self, login: str) -> Optional[str]:
         last_prs = self.__github.get_last_prs(login)
         last_prs_items = last_prs['items']
-        if len(last_prs_items) == 0:
-            return None
-        last_pr = last_prs_items[0]
-        created_at = last_pr['created_at']
-        return created_at
+        latest_date = None
+        for pr in last_prs_items:
+            latest_date = self.__get_max_date(pr['created_at'], latest_date)
+            latest_date = self.__get_max_date(pr['closed_at'], latest_date)
+            latest_date = self.__get_max_date(pr['updated_at'], latest_date)
+        if latest_date:
+            return datetime.strftime(latest_date, PR_DATE_FORMAT)
+        return None
+
+    def __get_max_date(self, date_str: Optional[str], latest_date: Optional[datetime]) -> Optional[datetime]:
+        if date_str is not None:
+            date = pr_date_str_to_date(date_str)
+            if latest_date is None or date > latest_date:
+                return date
+        return latest_date
+
+
+PR_DATE_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
```

Comment:
Small nit but you can move `PR_DATE_FORMAT` to the top of the file rather than in the middle.

Reply:
Code updated.